### DISCUSSION TITLE
Announce list sorting/filtering options

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [markqvist/Reticulum#407] [Nomad Network]
Sorting options for the Announce list would be really nice as the current mess isn't that easy to look through.
Some basic sorting options could be to only show nodes, peers, or both and to show any combination of trusted, unknown, and untrusted connections.

![image](https://github.com/markqvist/Reticulum/assets/105567407/dd775af8-9899-4aaa-9640-442d0b4f857b)

Thank you for the work you've done, this stuff is really cool.

**Comment by faragher** [comment]
Ah, good! I was a little more broad on your other post since I wasn't sure if you were on Sideband or NomadNet. The announce stream is chronologically sorted. You might be happier with the node list, which you can access with Ctrl+L. The hotkey list in the status bar at the bottom is a little unclear with Nodes/Announces, but I think that's the window you'd prefer for finding known nodes.

Now, if you're looking for a list of unsaved nodes that are sorted and peers you haven't had a conversation with, that's different. I'm uncertain that's what the screen is intended for, but Mark should be back after the holidays to weigh in on the intent and discuss options.

**Comment by LinuxinaBit** [comment]
> Ah, good! I was a little more broad on your other post since I wasn't sure if you were on Sideband or NomadNet.

I'm on both, this one is just specifically for NomadNet

> Now, if you're looking for a list of unsaved nodes that are sorted and peers you haven't had a conversation with, that's different.

That's what I'm looking at. I know how everything is laid out, and I like it. I just want a way to filter the Announce Stream specifically so I can, for example, quickly find nodes I haven't looked at yet (in this example one would filter to only show unsaved nodes).

**Reply by faragher** [reply]
I get your thinking now. Seems interesting and useful. Worth looking into.
